src/com/company/Controller/Command/Help.java

```diff
@@ -0,0 +1,42 @@
+package com.company.Controller.Command;
+
+import com.company.model.DatabaseManager;
+
+import java.sql.SQLException;
+
+/**
+ * Created by yulia on 08.11.16.
+ */
+public class Help extends AbstractCommand {
+
+    public Help() {
+    }
+
+    public Help(DatabaseManager databaseManager) {
+        super(databaseManager);
+    }
+
+    public static final String INFO = "All available command: " +
```

Comment:
if this constant is not used in another classed it can have a **private** modifier.